homeassistant/components/reolink/host.py

```diff
@@ -158,10 +158,10 @@ async def async_init(self) -> None:
         store: Store[str] | None = None
         if self._config_entry_id is not None:
             store = get_store(self._hass, self._config_entry_id)
-            if self._config.get(CONF_SUPPORTS_PRIVACY_MODE):
-                data = await store.async_load()
-                if data:
-                    self._api.set_raw_host_data(data)
+            if self._config.get(CONF_SUPPORTS_PRIVACY_MODE) and (
```

Comment:
you can walrus the store too and collapse another level 😄

Reply:
But I also need the store in a block later on, will that still work?

Reply:
Yeah it's just s variable assignment and truthy check in one.